src/config.ts

```diff
@@ -169,16 +176,39 @@ export class Configuration {
   private _loadFromEnvironment(): { [key: string]: any } {
     const conf: { [key: string]: any } = {}
 
-    Object.entries(ENV_TO_KEY_MAPPING).forEach(([k, v]) => {
+    STRING_KEYS.forEach(k => {
       const current = process.env[k]
 
       if (current) {
-        try {
-          // attempt to extract a value from a string
-          conf[v] = eval(current)
-        } catch (e) {
-          conf[v] = current
-        }
+        conf[ENV_TO_KEY_MAPPING[k]] = current
+      }
+    })
+
+    BOOL_KEYS.forEach(k => {
+      const current = process.env[k]
+
+      if (current) {
+        conf[ENV_TO_KEY_MAPPING[k]] = current == "true"
+      }
```

Comment:
Perhaps use the same logic here as in `LIST_OR_BOOL_KEYS`? Meaning, if it's not either `true` or `false`, don't set it.

That is what we do [in the Python instrumentation](https://github.com/appsignal/appsignal-python/blob/main/src/appsignal/config.py#L311-L321).

```suggestion
      if (current == "true") {
        conf[ENV_TO_KEY_MAPPING[k]] = true
      } else if (current == "false") {
        conf[ENV_TO_KEY_MAPPING[k]] = false
      }
```